features/media-transform.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-09-09
-name: MediaStreamTrack Insertable Media Processing using Streams
-description: TODO
+name: MediaStreamTrack insertable media processing using streams
+description: The MediaStreamTrack Insertable Media Processing using Streams API, also known as MIMPuS, gives direct access to media streams for manipulations such as merging streams, real time effects, and content overlays.
```

If the spec is going to lead with funny hats, then I think we should too:

```suggestion
description: The `MediaStreamTrackProcessor` and `MediaStreamTrackGenerator` APIs modify the contents of a media track for merging streams, applying real-time effects like funny hats, and content overlays.
```